frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/LookAndFeelSettings.tsx

```diff
@@ -180,26 +177,19 @@ export const LookAndFeelSettings = ({
           />
 
           {canWhitelabel && (
-            // We only show the "Download Data" toggle if the users are pro/enterprise
-            <DisplayOptionSection
-              title={t`Download data`}
-              titleId={downloadDataId}
-            >
-              <Switch
-                aria-labelledby={downloadDataId}
-                label={t`Enable users to download data from this embed`}
-                labelPosition="left"
-                size="sm"
-                variant="stretch"
-                checked={displayOptions.downloads ?? true}
-                onChange={e =>
-                  onChangeDisplayOptions({
-                    ...displayOptions,
-                    downloads: e.target.checked,
-                  })
-                }
-              />
-            </DisplayOptionSection>
+            <Switch
+              label={t`Download buttons`}
+              labelPosition="left"
+              size="sm"
+              variant="stretch"
+              checked={displayOptions.downloads ?? true}
```

Comment:
I don't like the fact that this needs require us providing the default value when used

https://github.com/metabase/metabase/blob/d2806babbe90f2ddfdc07457581c137379505da8/frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/config.ts#L12

But I kind of remember you tried fixing this, but it requires more effort so that's fine since it doesn't change from the previous version either. You just removed the extra label.

We should probably do something about the default value to keep it consistent later.